pkg/registry/pod/etcd/etcd.go

```diff
@@ -321,7 +322,7 @@ func (r *AttachREST) Connect(ctx api.Context, name string, opts runtime.Object)
 	if err != nil {
 		return nil, err
 	}
-	return genericrest.NewUpgradeAwareProxyHandler(location, transport, true), nil
+	return genericrest.NewUpgradeAwareProxyHandler(location, transport, false, true), nil
```

Comment:
good catch... looks like the local function only adds a bandwidth limiter. it was added in https://github.com/kubernetes/kubernetes/commit/99b02bfe and applies equally to attach